2015/presentations/with modern_peripherals: Python and Flask Piotr Dyba/text.md

```diff
@@ -0,0 +1,208 @@
+# with modern_peripherals: Python and Flask - Piotr Dyba
+
+##Introduction
+
+Auto-scrolling sites, glance-following ads, and gesture friendly web pages are 
+coming! Over the last few years three products emerged that enable interaction 
+with computer in a new way: Myo Armband, Leap Motion Controller and EyeTribe. 
+Using web sockets it is possible to interact using this new devices with web 
+apps and devices in completely new way. Lets start with really short 
+introduction to Web Sockets, after that I’ll get through implementation, 
+wrappers and an imaginative app for each of the new devices.
+
+##Web Sockets and Python socket solution
+
+In short, web sockets is a protocol providing full-duplex communication channels 
+over a single TCP connection available in web browsers 
+
+![Alt text](./001_web_sockets.png "01 Web Sockets Diagram")
+
+It starts with handshake which upgrades the HTTP connection to use the Web Socket protocol.
```

Comment:
Seems some words are missed here, also some typos.

How about: 
It starts with a handshake which upgrades the HTTP connection to use the Web Sockets protocol. This enables message exchanges from and to a server. Such exchange can be triggered by a user or by some programmer defined event. It's what interests us most. Communication channel is terminated after one of the sides sends a kill message.